sandbox/Generated/UnitGenerator/UnitGenerator.SourceGenerator/FileGenerate.A.Generated.cs

```diff
@@ -17,14 +12,13 @@ public A(int value)
         {
             this.value = value;
         }
-
-
-        public static explicit operator int(A value)
+        
+        public static implicit operator int(A value)
         {
             return value.value;
         }
 
-        public static explicit operator A(int value)
+        public static implicit operator A(int value)
```

Comment:
what the reason of different result, implicit and explicit?

Reply:
Oh.. simply my mistake... fixed.